packages/node/package.json

```diff
@@ -101,7 +101,8 @@
     "import-in-the-middle": "^1.13.0"
   },
   "devDependencies": {
-    "@types/node": "^18.19.1"
+    "@types/node": "^18.19.1",
+    "fastify": "4.18.0"
```

Comment:
Can we avoid this and just vendor in the minimal interface we need to satisfy the types?

Reply:
Sure 👍